views/default/css/admin.php

```diff
@@ -1300,42 +1300,43 @@
 /* ***************************************
 	ICONS
 *************************************** */
+.fa {
```

Comment:
i'll look in to removing all .fa css from the core files and replace it with .elgg-icon